OpenLIFUData/OpenLIFUData.py

```diff
@@ -2066,7 +2066,7 @@ def remove_protocol(self, protocol_id:str) -> None:
         protocolConfigLogic.delete_protocol_from_cache(protocol_id)
 
     def load_transducer_from_file(self, filepath:str) -> None:
-        transducer = openlifu_lz().Transducer.from_file(filepath)
+        transducer = openlifu_lz().xdc.util.load_transducer_from_file(filepath, convert_array=True)
```

Comment:
in the openlifu update just now, v0.9.0,, some transducers are read using `Transducer.from_file` while others are read using `TransducerArray.from_file`. The utility `openlifu.xdc.util.load_transducer_from_file` smartly figures out which one to use, hence we must now use it here.

(`TransducerArray` is a new concept that Peter introduced to simplify having to tweak things for batches of transducer elements at a time)